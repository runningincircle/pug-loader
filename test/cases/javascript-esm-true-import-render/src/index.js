import html1 from 'Views/widget.pug';
import html2 from 'Views/widget.pug?{"a":10,"b":"xyz"}';
import html3 from 'Views/widget.pug?a=20&b=zzz';

console.log(html1 + html2 + html3);