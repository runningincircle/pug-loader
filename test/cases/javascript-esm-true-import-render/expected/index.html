<h1>Hello!</h1><span>Param[a]: 7</span><span>Param[b]: zzz</span><img src="assets/images/image.697ef306.png"><img src="assets/images/image2.697ef306.png"><h1>Hello!</h1><span>Param[a]: 10</span><span>Param[b]: xyz</span><img src="assets/images/image.697ef306.png"><img src="assets/images/image2.697ef306.png"><h1>Hello!</h1><span>Param[a]: 20</span><span>Param[b]: zzz</span><img src="assets/images/image.697ef306.png"><img src="assets/images/image2.697ef306.png">